src/Apps/Order/Components/AddressVerificationFlow.tsx

```diff
@@ -100,22 +104,151 @@ const fallbackFromFormValues = (address: AddressValues): AddressOption => {
   }
 }
 
+const useAddressVerificationTracking = () => {
```

Comment:
created this to DRY up some repeated verification calls and remove hooks which were only needed for tracking.